### PR TITLE
no mock login in QA

### PR DESCRIPTION
**Pull request by YJouhar47**
https://binnenland.atlassian.net/browse/CLBV-765

### CODE REVIEW

File: app/router.js [hunk 1]
```diff
@@ -20,7 +20,12 @@ Router.map(function () {
     this.route('login');
     this.route('logout');
     if (ENV.controllerLogin !== 'true') {
-      this.route('switch');
+      if (ENV.acmidm.clientId === '{{OAUTH_API_KEY}}') {
+        this.route('mock-login');
+      }
+      this.route('switch-login');
+    } else {
+      this.route('controller-login');
```
Comment (aliokan) [inline app/router.js:28]: **Question:** Is it normal that login routes are defined twice, in `/authorization` and `/` level?

https://github.com/lblod/frontend-contactgegevens-loket/blob/98042a4f93333547650fa1eabf716b48fdb6b539/app/router.js#L9-L14

**Review by aliokan — COMMENTED** [review]
@YJouhar47 I left a question

File: app/router.js (hunk 1 above)
Comment (YJouhar47) [reply app/router.js:28]: yes, we defined login for the index page and a login for authorization of acm/idm

**Review by aliokan — APPROVED** [review]
:rocket:

**Pull request merged**